streampipes-client-python/streampipes/model/resource/query_result.py

```diff
@@ -71,8 +71,11 @@ def convert_to_pandas_representation(self) -> Dict[str, Union[List[str], List[Li
 
     total: StrictInt
     headers: List[StrictStr]
-    all_data_series: List[DataSeries]
+    all_data_series: List[DataSeries] = Field(alias="allDataSeries")
```

Comment:
Alias is not needed here (same for the other three fields) 
We have default alias generator in place, that creates the camel case name as alias for every field

Reply:
Ok I removed the added alias.